citest/json_predicate/logic_predicate.py

```diff
@@ -112,6 +118,9 @@ def __init__(self, pred):
   def __str__(self):
     return 'NOT ({0})'.format(self.__pred)
```

Comment:
You can force the `__str__` representation by changing this to `{0!s}`. I think that's the default, but it might be clearer since you're forcing the `__repr__` representation below.